kura/test/org.eclipse.kura.container.orchestration.provider.test/src/test/java/org/eclipse/kura/container/orchestration/provider/EnforcementSecurityTest.java

```diff
@@ -0,0 +1,254 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2024 Eurotech and/or its affiliates and others
```

Comment:
Since this is a new file it should only be 2024

```suggestion
 * Copyright (c) 2024 Eurotech and/or its affiliates and others
```